core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -309,42 +256,35 @@ public OutputStream create(String path) throws IOException {
    * @return A {@code OutputStream} object
    * @throws IOException if a non-Alluxio error occurs
    */
-  public OutputStream create(String path, CreateOptions options) throws IOException {
-    if (!options.isEnsureAtomic()) {
-      return createDirect(path, options);
-    }
-    return new NonAtomicFileOutputStream(path, options, this);
-  }
+  OutputStream create(String path, CreateOptions options) throws IOException;
 
   /**
-   * Creates a file in the under file system with the specified {@link CreateOptions}. This stream
-   * writes directly to the underlying storage without any atomicity guarantees.
+   * Deletes a directory from the under file system with the indicated name non-recursively.
    *
-   * @param path the file name
-   * @param options the options for create
-   * @return A {@code OutputStream} object
+   * @param path of the directory to delete
+   * @return true if directory was found and deleted, false otherwise
```

Comment:
Yes.  A non-recursive delete only succeeds on an empty directory. Updating the comment